packages/composites/dydx-rewards/src/method/formulas/dip-7.ts

```diff
@@ -0,0 +1,38 @@
+import * as bn from 'bignumber.js'
+import { AddressRewards, OracleRewardsData } from '../../ipfs-data'
+import { addReward } from '../poke'
+
+export const calcTraderRewards = (
+  epochData: OracleRewardsData,
+  addressRewards: AddressRewards,
+  traderRewardsAmount: bn.BigNumber,
+  a = 0.67, // TRADER_SCORE_A=670000000000000000
+  b = 0.28, // TRADER_SCORE_B=280000000000000000
+  c = 0.05, // TRADER_SCORE_C=050000000000000000
+): void => {
+  const traderScore: { [address: string]: bn.BigNumber } = {}
+  let traderScoreSum = new bn.BigNumber(0)
+
+  Object.keys(epochData.tradeFeesPaid).forEach((addr) => {
+    const f = epochData.tradeFeesPaid[addr]
+    const d = epochData.openInterest[addr] || 0
+    const g = epochData.averageActiveStakedDYDX[addr] || 0
```

Comment:
I might be missing something here: are these the raw `OracleRewardsData` values? What I noticed is that these values are divided by the `totalTradeFeesPaid F`, `openInterest D` like in your previous formula [here](https://github.com/smartcontractkit/external-adapters-js/blob/7754a9581f6eed8156a697dd474c6c2f67126db3/packages/composites/dydx-rewards/src/method/formulas/initial.ts#L108)